charts/templates/opensearch-operator-system-namespace.yaml

```diff
@@ -2,5 +2,4 @@ apiVersion: v1
 kind: Namespace
```

Comment:
The namespace object should be deleted completely from the helm chart. Either the user uses an existing namespace or `helm install --create-namespace`.

Reply:
deleted